server/public/userTestUpdate.html

```diff
@@ -54,7 +54,7 @@ <h3>Chess Room user details</h3>
       console.log(organization.value.toString());
       console.log(wins.value.toString());
       console.log(id.value.toString());
-      const url = `https://chess.zuri.chat/api/dbUpdate/${id.value}`
+      const url = `http://localhost:5050/api/dbUpdate/${id.value}`
```

Comment:
fixxx

Reply:
> fixxx thiss

done - I am using the localhost

Reply:
You cannot use localhost on the live server boss 🙂